evm/wallet.js

```diff
@@ -0,0 +1,18 @@
+const { ethers } = require('ethers');
+// console.log("random,", ethers.Wallet.createRandom().mnemonic)
+// Replace this seed with your own for deterministic behavior
+const seed = 'axelar dust wallet random private key generation using ethers library for use';
```

Comment:
Let's not hardcode mnemonics/seeds. Also pass them via env vars. Otherwise they may get accidentally checked-in.